.github/workflows/format-test-build.yml

```
@@ -0,0 +1,97 @@
+            UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
+            UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
+            UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
+          with:
+            targetPlatform: ${{ matrix.targetPlatform }}
+            buildName: SummitSeeker
+        - uses: actions/upload-artifact@v4
+          with:
+            name: Build-${{ matrix.targetPlatform }}
+            path: build/${{ matrix.targetPlatform }}
+
+    
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)